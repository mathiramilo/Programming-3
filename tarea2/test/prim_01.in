leer_grafo
3 no-dirigido
1: 2 p5 3 p3 .
2: 3 p4 .
3: .
0: .
prim
Fin