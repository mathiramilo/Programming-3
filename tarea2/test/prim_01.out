1>El grafo leído es:
Imprimiendo Grafo
Cantidad de vértices: 3
Cantidad de aristas: 3
1: (2,5)(3,3).
2: (1,5)(3,4).
3: (1,3)(2,4).
2>El costo del MST es: 7, y el MST:
Imprimiendo Grafo
Cantidad de vértices: 3
Cantidad de aristas: 2
1: (3,3).
2: (3,4).
3: (1,3)(2,4).
3>Fin.
